src/App.jsx

```diff
@@ -1,28 +1,48 @@
 import React from 'react';
 import './App.scss';
 import './styles/general.scss';
-import { TodoList } from './components/TodoList';
-import { CurrentUser } from './components/CurrentUser';
+import CurrentUser from './components/CurrentUser/CurrentUser';
+import TodoList from './components/TodoList/TodoList';
+import { getTodos } from './api/api';
 
 class App extends React.Component {
   state = {
     todos: [],
-    selectedUserId: 0,
+    selectedUserId: null,
   };
 
+  async componentDidMount() {
+    this.setState({
+      todos: await getTodos(),
+    });
+  }
+
+  setSelectedUser = (id) => {
+    this.setState({
+      selectedUserId: id,
+    });
+  }
+
   render() {
     const { todos, selectedUserId } = this.state;
 
     return (
       <div className="App">
         <div className="App__sidebar">
-          <TodoList todos={todos} />
+          <TodoList
+            todos={todos}
+            onChangeUser={this.setSelectedUser}
+            selectedUserId={selectedUserId}
+          />
         </div>
 
         <div className="App__content">
           <div className="App__content-container">
             {selectedUserId ? (
-              <CurrentUser userId={selectedUserId} />
+              <CurrentUser
+                userId={selectedUserId}
+                onChangeUser={this.setSelectedUser}
```

Comment:
если тут ты только очищаешь то я бы сделал так
```suggestion
                onUserUnselected={() => this.setSelectedUser(null)}
```
И тогда внутри тебе не надо думать какое именно значение установить для очистки 0 или `null` например